notebooks/Add GPS Metadata Fields_FS.ipynb

```diff
@@ -0,0 +1,266 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    if (item.type != 'Feature Service'):
</pre><p>Should add a check for <code>if item is None raise Exception("Cannot find item")</code></p><p>The API will return None if it doesn't find an item, then the next line would raise an exception that wouldn't be super helpful</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/field-maps-scripts/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>